tutorials/Introduction.ipynb

```diff
@@ -191,96 +237,120 @@
+        "times = data.times\n",
+        "series = data.series\n",
+        "values = series[item_id] \n",
+        "\n",
+        "fig = go.Figure()\n",
+        "fig.add_trace(go.Scatter(x=times, y=values))\n",
+        "fig.update_layout(title=item_name)\n",
+        "fig.show()"
       ],
       "outputs": [],
       "metadata": {
-        "id": "x2Z6qa4AeZdx",
         "colab": {
           "base_uri": "https://localhost:8080/",
-          "height": 541
+          "height": 542
         },
-        "outputId": "01927fb0-3cf5-4027-a0b4-505345c703c3"
+        "id": "HG4UzxwEb_iM",
+        "outputId": "d7b32f96-60f7-41fb-b028-612e58a30362"
       }
     },
     {
       "cell_type": "markdown",
       "source": [
-        "### Writing data\n",
-        "Writing data is done in two step:\n",
-        "* Writing meta data\n",
-        "* Writing values\n",
+        "<a name=\"write\"></a>\n",
+        "### Writing data back to Clarify\n",
+        "Now that we have imported an Item to Notebook, it's time to send data back to Clarify.\n",
         "\n",
-        "Writing meta data can be done with creating a `Signal` data type and populating it with meta data. The ID of this signal needs to correspond with the ID we use for writing value to it.\n",
+        "Writing data to Clarify is done in two steps:\n",
+        "* Create a new `Signal`\n",
+        "* Add data to the new `Signal`\n",
         "\n",
-        "#### Creating a new Signal\n",
-        "We will do a simple rolling window to create a new Item in the Clarify app. First we want to create the meta data for the Item with a `Signal` data type."
+        "Writing meta data can be done by creating a `Signal` and populating it with meta data. The ID of this `Signal` needs to correspond with the ID we use for writing values to it.\n",
+        "\n",
+        "<img src=\"https://raw.githubusercontent.com/searis/data-science-tutorials/update_ui/media/introduction/light-2.png\" alt=\"clarify doodle\" width=\"400\">\n",
+        "\n",
+        "#### Create a new Signal\n",
+        "The new `Signal` will contain a simple rolling window based on the `Item` we visualized above. First we want to create the meta data for the `Signal` with a `Signal` data structure.\n",
+        "\n",
+        "> *Why do we have both `Signals` and `Items`?*<br>\n",
+        "> Signals map to the raw sensor that they are recieving data from. They are supposed to be a 1 to 1 mapping in the `Signals` meta data. `Items` is an abstraction of the `Signal`. The `Item` can have custom meta data and even consist of several `Signals`. \n",
+        ">\n",
+        "> *Why would you connect several `Signals` to an `Item`?* <br>\n",
+        "> You might change sensors, or even connect a new one to an `Item`. To keep the historical values you can also connect several `Signal`. Clarify will even support *Calculated Items* in the future, which is aggregated from a combination of one or more items.   "
```

Comment:
"Clarify will even support Calculated Items in the future". Maybe we can just say that it does since it will be true in a couple of days?

Reply:
Yes, but I want to link to some source where they can read more about it. So I will update once that is true